code/modules/projectiles/ammo_datums.dm

```diff
@@ -2594,11 +2598,10 @@ datum/ammo/bullet/revolver/tp44
 /datum/ammo/flamethrower/blue
 	name = "blue flame"
 	hud_state = "flame_blue"
-	max_range = 6
+	max_range = 7
```

Comment:
Uh, why is this in this pr ?

Reply:
This change exclusively affects use of the ammo datum as a projectile (same with the damage change). The other nozzles operate off the gun's max_range for determining how long of a line to ignite and won't be affected by any of the ammo datum changes unless you touch burn_time and burn_level, since they don't actually fire projectiles.